packages/automated-dispute/tests/services/eboActor/onLastBlockupdated.spec.ts

```diff
@@ -14,8 +14,9 @@ describe("EboActor", () => {
             const request = DEFAULT_MOCKED_REQUEST_CREATED_DATA;
             const { disputeModuleData } = request.decodedData;
 
-            const responseToSettle = mocks.buildResponse(request, { id: "0x10" as ResponseId });
-
+            const responseToSettle = mocks.buildResponse(request, {
+                id: ("0x" + "02".repeat(32)) as ResponseId,
```

Comment:
I think you could also use viem's `pad` [function](https://viem.sh/docs/utilities/pad.html); just wanted to mention it!
```suggestion
                id: pad("0x02") as ResponseId,
```